### DISCUSSION TITLE
Error when building new scafoolded module

### DISCUSSION THREAD

**Discussion by zoltanhalasz** [oqtane/oqtane.framework#1016] [Q&A]
I have Oqtane 2.00 and when I generate a new developer module, called "CreatedModule", it won't build its projects - after starting the new solution and building it, gives the following error:

Error	CS0234	The type or namespace name 'CreatedModule' does not exist in the namespace 'CreatedModule.CreatedModule' (are you missing an assembly reference?)	CreatedModule.CreatedModule.Client	C:\Dotnet\CreatedModule.CreatedModule\Client\Edit.razor	2	Active

**Comment by hishamco** [comment]
I may try this, but one question does the namespace in your case same as module name? or it's generated?

**Reply by zoltanhalasz** [reply]
It was a generated external module, but the namespace and module name are the same, so it is always called "MyProject.MyProject" for any new module created. But whenever I compile, it's an error as described above.

**Reply by zoltanhalasz** [reply]
Ok I figured it out. The oqtane module has to be named correctly, like Owner.ProjectName

**Reply by sbwalker** [reply]
Validation added in #1089 to prevent this issue
